index.html

```diff
@@ -476,7 +479,13 @@ <h2>Installation</h2>
         <b>Bower</b> <tt>bower install underscore</tt>
       </li>
       <li>
-        <b>Component</b> <tt>component install jashkenas/underscore</tt>
+        <b>ExtendScript</b> <tt>#include "underscore.js"</tt>
+      </li>
+      <li>
+        <b>Rollup</b> If you want to enable treeshaking and you don&rsquo;t
```

Comment:
Is this just for Rollup, or for any ESM treeshaking bundler? (i.e. will these instructions work for Webpack too?)

Reply:
To be honest, I don't know. I try to avoid Webpack at all cost.